app/views/idv/in_person/bar_code.html.erb

```diff
@@ -0,0 +1,68 @@
+<% title t('forms.backup_code.title') %>
+
+<h5 class="my1 caps bold accent-blue">
+  <%= t('in_person_proofing.step', step: 3) %>
+</h5>
+
+<br/>
+
+<h1 class="h3 my0">
+  <%= t('in_person_proofing.headings.enrollment_form') %>
+</h1>
+<p class="mt-tiny mb3">
+  <%= t('in_person_proofing.info.enrollment_form') %>
+</p>
+
+<div class="mb4">
+  <fieldset class="m0 p0 rounded-xl">
+    <ul class="list-reset">
+      <li class="px-12p mb1 border-none mt1 m2">
+        <div class="inline-block h1 bold">
+          <%= t('in_person_proofing.enrollment_form') %>
+        </div>
+        <div class="h5 mt0 mb0 pt1 pb2 fs-12p">
+          <%= t('users.backup_code.generated_on_html',
+                date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y'))) %>
```

Comment:
something we could consider is naming this format in this YML and updating it to be more appropriate for other languages: https://github.com/18F/identity-idp/blob/master/config/locales/time/en.yml